### DISCUSSION TITLE
What is AlphaSynthWebAudioOutput in the official guides?

### DISCUSSION THREAD

**Discussion by TheLucifurry** [CoderLine/alphaTab#895] [Q&A] [answered]
`AlphaSynthWebAudioOutput`
I just can't find how to use it and where it's from in this manual)
https://www.alphatab.net/docs/guides/lowlevel-apis#playing-files-via-alphasynth

**Comment by Danielku15** [comment, marked as answer]
> What is AlphaSynthWebAudioOutput in the official guides?

It is the component responsible for the playback of the synthesized audio samples. The 

> I just can't find how to use it and where it's from in this manual)

In 1.2 we made some more parts of alphaTab internal and it seems we accidentally made the `AlphaSynthWebAudioOutput` only available internally. I added https://github.com/CoderLine/alphaTab/issues/897 to make the related classes public again and update the guide accordingly.

**Comment by andre-lima** [comment]
Hello @Danielku15 was this reverted somehow? I still can't find where to get this class from. Even a search on the code base only seems to return `AlphaSynthWebAudioOutputBase`.

Is this the correct one? `AlphaSynthAudioWorkletOutput`

**Reply by Danielku15** [reply]
Here you can see how we create an alphaSynth instance: 
https://github.com/CoderLine/alphaTab/blob/develop/src/platform/javascript/BrowserUiFacade.ts#L501-L544 

With the addition of AudioWorklets we do some checks which variant to create exactly. The `AlphaSynthWebWorkerApi` puts the audio generation into a web worker, and the output decides how to play the samples.
